benchopt/runner.py

```diff
@@ -382,6 +492,10 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
                             force=force, pdb=pdb
                         ))
     df = pd.DataFrame(run_statistics)
+    # print(df)
+    df = cumsum_time(df)
```

Comment:
I don't get why you would do this?

Reply:
We need the cumulative time instead of the time of each iteration I guess

Reply:
No, each entry is the complete timing as the computation so it should not be summed.